include/contractor/query_graph.hpp

```diff
@@ -15,7 +15,7 @@ namespace detail
 {
 template <storage::Ownership Ownership>
 using QueryGraph = util::StaticGraph<typename QueryEdge::EdgeData, Ownership>;
-}
+} // namespace detail
```

Comment:
clang-tidy has a check which fires if we skip it somewhere, I fixed some of them, but didn't enable it, because there are tons of such places and probably worth doing it in separate PR